### PR TITLE
Update with 2 new events + added a footer

### PR DESCRIPTION
**Pull request by Joniff**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,13 +36,16 @@
 					<div id="contents" class="left">
 						<h2><span>Upcoming Events</span></h2>
 						<ul>
-							<li>No upcoming events have been planned. If you would like to organise an umBristol event, <a href="http://www.meetup.com/messages/?new_convo=true" target="_blank">please get in touch</a>.</li>
+							<!--<li>No upcoming events have been planned. If you would like to organise an umBristol event, <a href="http://www.meetup.com/messages/?new_convo=true" target="_blank">please get in touch</a>.</li>-->
+							<li style="color:gray;"><a href="javascript:;" target="_blank">umBristol BBQ</a> - 3:30pm to Late, Friday 14<sup>th</sup> June 2017 @ TBC</li>
```
Comment (leekelleher) [inline index.html]: The CSS has a class called `idea` (in the `li` tag) if you wanna use that?

File: index.html [hunk 2]
```diff
@@ -36,13 +36,16 @@
...
+							<li><a href="https://www.meetup.com/umBristol/events/240574745/" target="_blank">umBristol Talks</a> - 6:30pm, Wednesday 5<sup>th</sup> June 2017 @ <a href="https://goo.gl/maps/yCdTJ82ZoaG2" target="_blank">Rock Solid Knowledge</a></li>
```
Comment (leekelleher) [inline index.html]: July, not June?

Damn, I can't make that one, (I'll be playing in my band).

Comment (Joniff) [reply index.html]: lol, yeah, they all have the wrong month

**Comment by Joniff** [comment]
OK have updated to use idea class and now the forthcoming events are in July

**Comment by leekelleher** [comment]
Aces, thanks @Joniff MVP!

**Pull request merged**